modules/hrm/includes/class-announcement-list-table.php

```diff
@@ -31,23 +33,281 @@ public function no_items() {
      * @return array
      */
     public function get_columns() {
-        $columns    =   [
+        $columns = [
             'cb'        => '<input type="checkbox" />',
             'title'     => __( 'Title', 'erp' ),
-            'sent_to'   => __( 'Sent to', 'erp' ),
             'type'      => __( 'Type', 'erp' ),
+            'sent_to'   => __( 'Sent to', 'erp' ),
             'date'      => __( 'Date', 'erp' ),
         ];
 
         return $columns;
     }
 
+    /**
+     * Default column values if no callback found
+     *
+     * @since 1.9.1
+     *
+     * @param object $item
+     * @param string $column_name
+     *
+     * @return string
+     */
+    public function column_default( $item, $column_name ) {
+        switch ( $column_name ) {
+            case 'title':
+                return esc_html( $item->post_title );
+
+            case 'type':
+                $type = ucfirst( get_post_meta( $item->ID, '_announcement_type', true ) );
+                return ( empty( $type ) ? '&mdash;' : esc_html( $type ) );
+
+            case 'date':
+                return erp_format_date( $item->post_date );
+
+            default:
+                return isset( $item->$column_name ) ? $item->$column_name : '';
+        }
+    }
+
+    /**
+     * Render checkbox column
+     *
+     * @since 1.9.1
+     *
+     * @param Object $item
+     *
+     * @return void
+     */
+    public function column_cb( $item ) {
+        return sprintf( '<input type="checkbox" name="announcement_ids[]" value="%d" />', $item->ID );
+    }
+
+    /**
+     * Render the name column
+     *
+     * @since 1.9.1
+     *
+     * @param Object $item
+     *
+     * @return void
+     */
+    public function column_title( $item ) {
+        $actions = [];
+
+        $params = [
+            'post' => $item->ID,
+        ];
+
+        $params['action'] = 'edit';
+        $edit_text        = esc_html__( 'Edit', 'erp' );
+        $edit_title       = esc_html__( 'Edit this item', 'erp' );
+
+        if ( ! empty( $_GET['status'] ) && sanitize_text_field( wp_unslash( $_GET['status'] ) ) === 'trash' ) {
+            $params['action']   = 'untrash';
+            $params['_wpnonce'] = wp_create_nonce( 'untrash-post_' . $item->ID );
+            $edit_text          = esc_html__( 'Restore', 'erp' );
+            $edit_title         = esc_html__( 'Restore this item', 'erp' );
+        }
+
+        $edit_url = add_query_arg( $params, admin_url( 'post.php' ) );
+
+        $params['action']   = ( ! empty( $_GET['status'] ) && sanitize_text_field( wp_unslash( $_GET['status'] ) ) === 'trash' ) ? 'delete' : 'trash';
```

Comment:
You're using this `sanitize_text_field( wp_unslash( $_GET['status'] ) )` multiple times (previously on line#107). So first, store it in a variable, then use the variable where needed.